expat/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ option(BUILD_examples "build the examples for expat library" ON)
 option(BUILD_tests "build the tests for expat library" ON)
 option(BUILD_shared "build a shared expat library" ON)
 option(BUILD_doc "build man page for xmlwf" ${BUILD_doc_default})
+option(BUILD_fuzz "build fuzzers for the expat library" OFF)
```

Comment:
Is name `fuzz` a convention that is widely followed? My vote for `fuzzers` over `fuzz` for a name otherwise. In folder names as well maybe?